src/development.controller.ts

```diff
@@ -105,9 +103,9 @@ export class DevelopmentController {
   }
 
   @Post('scan/:blockNumber')
-  async scanChainFromBlock(@Param('blockNumber') blockNumber: string) {
-    const block = BigInt(blockNumber) - 1n;
-    await this.cacheManager.set(LAST_SEEN_BLOCK_NUMBER_KEY, block.toString());
+  async scanChainFromBlock(@Param('blockNumber') blockNumber: number) {
+    const block = blockNumber - 1;
+    await (this.scannerService as unknown as any).setLastSeenBlockNumber(block);
```

Comment:
Why the casting?

Reply:
Only thing I would add is will number fit bigger block numbers?

Reply:
It's actually a protected method, and I didn't really want to make it public just to support the "development" controller. I wanted to be clear that this is NOT to be done normally 😅

Reply:
Also, block numbers are `u32`